src/sdk/analytics/platform/vtex/search.ts

```diff
@@ -0,0 +1,68 @@
+/**
+ * More info at: https://www.notion.so/vtexhandbook/Event-API-Documentation-48eee26730cf4d7f80f8fd7262231f84
+ */
+
+import config from 'src/../store.config'
+
+const THIRTY_MINUTES_S = 30 * 60
+const ONE_YEAR_S = 365 * 24 * 3600
```

Comment:
```suggestion
const ONE_YEAR_S = 365 * 24 * 3_600
```

Reply:
Or you can just multiply 60 secs * 60 min

Reply:
it's so much clearer for me 3600